src/models/train_model.py

```diff
@@ -269,14 +225,11 @@ def initialize_and_run(data_model, features, lm_features, config_level,
 
     # Initialize tuner
     tune = Tuner(df)
-    try: 
-        # Base XG model
-        tune.tune('XG_base', 'XGBClassifier', features, cvp, mp['XGBClassifier'])
-        # Base LR model
-        tune.tune('LR_base', 'LogisticRegression', lm_features, cvp, mp['LogisticRegression'])
-    except ValueError:
-        print('CV fails, likely very few of target available, try rerunning at segment-level')
-        raise
+    # Base XG model
```

Comment:
Do you think it makes sense to remove this try/except? It might still be the case that very few of the segments have crashes on them..

Reply:
Fair enough, I'll put it back